{"uuid": "3a315091-2911-40d1-b934-da0c3e41840c", "children": ["15af0948-f7f2-4d22-8e9b-18d0131c7166"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidSessionIdException: Message: invalid session id\nStacktrace:\n\tGetHandleVerifier [0x0065D363+60275]\n\tGetHandleVerifier [0x0065D3A4+60340]\n\t(No symbol) [0x0049056E]\n\t(No symbol) [0x004CB8E8]\n\t(No symbol) [0x004FCF46]\n\t(No symbol) [0x004F8A18]\n\t(No symbol) [0x004F7F96]\n\t(No symbol) [0x004636E5]\n\t(No symbol) [0x00463C3E]\n\t(No symbol) [0x004640CD]\n\tGetHandleVerifier [0x008A8FE3+2467827]\n\tGetHandleVerifier [0x008A45E6+2448886]\n\tGetHandleVerifier [0x008BF80C+2560028]\n\tGetHandleVerifier [0x00673DF5+153093]\n\tGetHandleVerifier [0x0067A3BD+179149]\n\t(No symbol) [0x004633B0]\n\t(No symbol) [0x00462BC3]\n\tGetHandleVerifier [0x009CDFBC+3667916]\n\tBaseThreadInitThunk [0x76E3FCC9+25]\n\tRtlGetAppContainerNamedObjectPath [0x774282AE+286]\n\tRtlGetAppContainerNamedObjectPath [0x7742827E+238]\n\n", "trace": "  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\Desktop\\Ambition hire file\\Ambition hire\\conftest.py\", line 85, in driver\n    driver.implicitly_wait(15)\n  File \"C:\\Users\\Dell\\AppData\\Roaming\\Python\\Python312\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 654, in implicitly_wait\n    self.execute(Command.SET_TIMEOUTS, {\"implicit\": int(float(time_to_wait) * 1000)})\n  File \"C:\\Users\\Dell\\AppData\\Roaming\\Python\\Python312\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 348, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Dell\\AppData\\Roaming\\Python\\Python312\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1745925395530, "stop": 1745925404032}], "start": 1745925395530, "stop": 1745925408168}